pkgs/development/python-modules/pysmt/default.nix

```
@@ -1,28 +1,21 @@
 { lib
 , buildPythonPackage
-, fetchFromGitHub
-, nose
-, pytestCheckHook
-, six
+, fetchPypi
 }:
 
 buildPythonPackage rec {
   pname = "pysmt";
-  version = "0.9.0";
+  version = "0.9.1.dev132";
+  format = "wheel";
```

why the wheel?

---

Upstream is releasing the `.devX` versions only as wheels on PyPI. `dev` releases on GitHub are not tagged.

---

-_- sigh.

Can you add a comment?
```suggestion
  format = "wheel"; # dev versions are only distributed as wheels
```

---

Point taken

Request for tags: https://github.com/pysmt/pysmt/issues/693